src/bson.c

```diff
@@ -62,6 +66,9 @@
 
 #define PHONGO_ODM_FIELD_NAME "__pclass"
 
+#define IS_CLASS_INSTANTIATABLE(ce) \
```

Comment:
I can toss a `PHONGO` prefix on this, unless you think it's better to undefine at the bottom of the file. I currently have no use for this outside of `bson.c`.